<launch>
    <include file="$(find topnav_capo2)/launch/camera.launch">
        <arg name="camera_id" value="1"/>
        <arg name="camera_width" value="320"/>
        <arg name="camera_height" value="240"/>
    </include>
</launch>
